### DISCUSSION TITLE
utils 0.1.5 has breaking changes

### DISCUSSION THREAD

**Discussion by krutoo** [floating-ui/floating-ui#2588] [General]
Hi in my ui libarary package i use `@floating-ui/react` 0.25.4

When my package used in other project it causes error:

```
package path ./react is not exported from package <...>@floating-ui/utils <...>
```

This project uses Yarn and my library as dependency

Yarn adds 0.26.0 because my project uses `^0.25.4`

How can i solve this?

**Comment by atomiks** [comment]
I'm sorry but this is a mistake with the publishing. I will release a new version of `utils` that restores the `/react` path for `v0.1.x`.

**Reply by krutoo** [reply]
thanks a lot

**Reply by atomiks** [reply]
While that will fix the new issue, make sure all your `@floating-ui/react` versions are `v0.26.0` to pick up the latest versions/fixes though. Not sure what you mean by "Yarn adds 0.26.0 because my project uses ^0.25.4"?

**Reply by atomiks** [reply]
`@floating-ui/utils@0.1.6` is now released which fixes new installs that rely on `@floating-ui/react@0.25.x` since it restores the `/react` path. Sorry for the trouble.

However, `^0.25.4` will not install `0.26.x`, because in v0 that locks it to any latest `0.25.x` version, by semver. 

If you ensure all your versions are `@floating-ui/react@0.26.0`, then there will be no conflict or semver issues regardless of the change (and will ensure you don't have duplicates).
